api/types/autoupdate/constants.go

```diff
@@ -0,0 +1,29 @@
+package autoupdate
+
+const (
+	// ToolsUpdateModeEnabled enables client tools automatic updates.
+	ToolsUpdateModeEnabled = "enabled"
+	// AgentsUpdateModeEnabled enabled agent automatic updates.
+	AgentsUpdateModeEnabled
+	// ToolsUpdateModeDisabled disables client tools automatic updates.
+	ToolsUpdateModeDisabled = "disabled"
+	// AgentsUpdateModeDisabled disables agent automatic updates.
+	AgentsUpdateModeDisabled
+	// AgentsUpdateModeSuspended temporarily suspends agent automatic updates.
+	AgentsUpdateModeSuspended = "suspended"
```

Comment:
Not strongly opinionated, but I'd favor repeating the string constants and defining separate groupings for Tools vs. Agents. Too clever for me 🙂